test/network-settings-proxy.test.js

```diff
@@ -84,27 +87,27 @@ contract('Network Settings', accounts => {
     })
 
     it('has registration fee', async () => {
-      const fee = await ctx.contracts.settings.registrationFee.call()
+      const fee = await ctx.contracts.settings.getRegistrationFee.call()
```

Comment:
i would prefer not to use the get prefix, since a lot of our front end code would have to change.  But i'll leave that up to you.